cmd/cashierd/main.go

```diff
@@ -309,18 +314,82 @@ func certStore(config string) (store.CertStorer, error) {
 	return cstore, err
 }
 
+func dropPrivileges(runAsUser string) error {
```

Comment:
You'll only ever return `nil` from this function. You should replace `log.Fatal` calls with `return` and let the calling function handle the error.